src/System.Memory/ref/System.Memory.cs

```diff
@@ -227,7 +227,8 @@ public partial interface IMemoryList<T>
     {
         System.Memory<T> Memory { get; }
         System.Buffers.IMemoryList<T> Next { get; }
-        long RunningIndex { get; }
+        System.Buffers.IMemoryList<T> GetNext(long offset, out int localIndex);
+        long GetLength(System.Buffers.IMemoryList<T> memoryList);
```

Comment:
I think the PR is moving the type in the right direction functionality-wise, but this signature is strange.